pyvo/conftest.py

```diff
@@ -7,14 +7,19 @@
 
 """
 
+import numpy as np
```

Comment:
I think this numpy import can go inside the "if" ?

Reply:
Ah, nvm... I see that you using it for version compare, ignore me.

Reply:
It doesn't matter anyway as we import numpy as soon as we touch astropy.